src/test/java/uk/gov/pay/api/json/RequestJsonParserTest.java

```diff
@@ -160,6 +162,24 @@ public void parsePaymentRequest_whenDelayedCaptureFieldIsNotABoolean() throws Ex
         parsePaymentRequest(jsonNode);
     }
 
+    @Test
+    public void parsePaymentRequest_whenMotoFieldIsNotABoolean() throws Exception {
```

Comment:
this is already tested in `CreateCardPaymentRequestDeserializerTest. deserialize_shouldThrowValidationException_whenMotoIsNotABoolean`